swap_meet/clothing.py

```diff
@@ -1,2 +1,13 @@
-class Clothing:
-    pass
+from swap_meet.item import Item
+
+class Clothing(Item):
```

Comment:
It would be nice to put a blank line between function definitions, just for readability.  Otherwise well done.